modules/private-service-connect/variables.tf

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "Project ID for Private Service Connect."
+  type        = string
+}
+
+variable "network_self_link" {
+  description = "Network self link for Private Service Connect."
+  type        = string
+}
+
+variable "dns_code" {
+  description = "Code to identify DNS resources in the form of `dz-{dns_code}-{dns_type}`"
```

Comment:
I think we should not enforce dz prefix (we can default to it) but let users specify the whole prefix.

Reply:
Ok I will improve this input

Reply:
dns-code defaults to dz and new name composition is {dns-code}-{dns-type}